idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/test_column.py

```diff
@@ -0,0 +1,297 @@
+            "process_type": Fpar,
+            "finite_elements": x_nfe,
+            "length_domain_set": x_nfe_list,
+            "transformation_method": "dae.finite_difference",
+            "vapor_side": {
+                "transformation_scheme": "BACKWARD",
+                "property_package": m.fs.vapor_properties,
+                "has_pressure_change": False,
+                "pressure_drop_type": None},
+            "liquid_side":
+            {
+                "transformation_scheme": "FORWARD",
+                "property_package": m.fs.liquid_properties
+            }})
+
+        # Fix  input variables
+        m.fs.col.diameter_column.fix(0.64135)
+        if Fpar == ProcessType.absorber:
+            m.fs.col.length_column.fix(18.15)
+            # Stream exit temperature
+            m.fs.col.vapor_exit_temp = Param(initialize=346.17684550190495)
+            m.fs.col.liquid_exit_temp = Param(initialize=322.95279074281416)
+            for t in m.fs.time:
+                # Vapor
+                m.fs.col.vap_in_flow[t].fix(21.48)
+                m.fs.col.vap_in_temperature[t].fix(317.88)
+                m.fs.col.pressure_bottom[t].fix(107650)
+                m.fs.col.vap_in_mole_frac[t, "CO2"].fix(0.11453)
+                m.fs.col.vap_in_mole_frac[t, "H2O"].fix(0.08526)
+                m.fs.col.vap_in_mole_frac[t, "N2"].fix(0.73821)
+                m.fs.col.vap_in_mole_frac[t, "O2"].fix(0.06200)
+                # Liquid
+                m.fs.col.liq_in_flow[t].fix(37.55)
+                m.fs.col.liq_in_temperature[t].fix(319.87)
+                m.fs.col.liq_in_mole_frac[t, "CO2"].fix(0.00963)
+                m.fs.col.liq_in_mole_frac[t, "H2O"].fix(0.87435)
+                m.fs.col.liq_in_mole_frac[t, "MEA"].fix(0.11602)
+
+            # Initialize column
+            m.fs.col.initialize()
+        elif Fpar == ProcessType.stripper:
+            m.fs.col.length_column.fix(12.1)
+            # Stream exit temperature
+            m.fs.col.vapor_exit_temp = Param(initialize=396.51574931626186)
+            m.fs.col.liquid_exit_temp = Param(initialize=393.86554487492464)
+            for t in m.fs.time:
+                # Vapor
+                m.fs.col.vap_in_flow[t].fix(17.496)
+                m.fs.col.vap_in_temperature[t].fix(396.6)
+                m.fs.col.pressure_bottom[t].fix(183430)
+                m.fs.col.vap_in_mole_frac[t, "CO2"].fix(0.0145)
+                m.fs.col.vap_in_mole_frac[t, "H2O"].fix(0.9855)
+                # Liquid
+                m.fs.col.liq_in_flow[t].fix(84.48)
+                m.fs.col.liq_in_temperature[t].fix(382.15)
+                m.fs.col.liq_in_mole_frac[t, "CO2"].fix(0.0331)
+                m.fs.col.liq_in_mole_frac[t, "H2O"].fix(0.8547)
+                m.fs.col.liq_in_mole_frac[t, "MEA"].fix(0.1122)
+
+            # initialize column
+            m.fs.col.initialize(
+                homotopy_steps_h=[0.1, 0.2, 0.4, 0.6, 0.8, 1])
+        return m
+
```

Comment:
There needs to be a separate test for the default initialization scheme to make sure it works (even though it might be slow). You should use the `initilaization_tester` from `idaes.core.util.testing` to do this - see the core unit model tests for examples.

Reply:
@andrewlee94, the column did not pass this test. For now, the pressure balances are deactivated during initialization without being activated. These are included because there will be a need for them when the pressure drop is included in the nearest future. Aside from this, not sure why the original and final active constraints fail test here. 
````
>       assert len(fin_act_consts) == len(orig_act_consts)
E       AssertionError
```

Reply:
@PTA0001 The pressure balances will be the cause of the problem. At the simplest level - an initialize method should never change the final state of the model (fixed/unfixed, activated/deactivate).

For the case you are discussing here, it sounds like the pressure balances should either be:

1. Optional, controlled by a config argument so they are only built if needed (and then reactivated by the initialize method at the end).
2. Deactivated immediately after construction, so that they can be activated by the user if needed. Again, in this case the initialize method should check whether the pressure balances are active at the beginning, and then reactivate them as necessary at the end.